Dependency missing in installation from source and pytest segmentation fault

## ❓ Questions & Help
Hi, since I am interested in contributing to DeepChem, I followed the steps provided in the documentation to install DeepChem [from source with Conda](https://deepchem.readthedocs.io/en/latest/get_started/installation.html#from-source-with-conda). Then, to verify my installation was successful, I proceeded to run the tests. This is the output
```
~/python_playground/deepchem (master) » pytest -m "not slow" deepchem                                                                                                              estefania@BioMac
======================================================================================= test session starts ========================================================================================
platform darwin -- Python 3.8.13, pytest-7.1.2, pluggy-1.0.0
rootdir: /Users/estefania/python_playground/deepchem, configfile: setup.cfg
plugins: flaky-3.7.0, typeguard-2.13.3, cov-3.0.0
collected 870 items / 1 error / 34 deselected / 836 selected                                                                                                                                       

============================================================================================== ERRORS ==============================================================================================
___________________________________________________________ ERROR collecting deepchem/models/lightning/tests/test_dc_lightning_module.py ___________________________________________________________
deepchem/models/lightning/tests/test_dc_lightning_module.py:9: in <module>
    from deepchem.models.lightning.dc_lightning_module import DCLightningModule
deepchem/models/lightning/__init__.py:2: in <module>
    from deepchem.models.lightning.dc_lightning_module import DCLightningModule
deepchem/models/lightning/dc_lightning_module.py:9: in <module>
    class DCLightningModule(pl.LightningModule):
E   NameError: name 'pl' is not defined
```

Checking at `deepchem/models/lightning/dc_lightning_module.py`, seems like the dependency `pytorch-lightening` is missing.
I proceeded to install the missing dependency with `pip install pytorch-lightning`. 

After installation, when running `pytest -m "not slow" deepchem` collection of all tests works as expected. However, there is a segmentation fault in `deepchem/feat/tests/test_smiles_tokenizer.py`, as shown in the image below:

<img width="1378" alt="image" src="https://user-images.githubusercontent.com/27988163/180037056-f1ad25a5-0d24-4b74-adf9-4fb7b16e1bf6.png">

Questions:
- Shouldn't be `pytorch-lightning` included in `requirements/torch/env_torch.mac.cpu.yml`?
- Would you please point to the workflow that runs the CI tests on macOS? I only found [this](https://github.com/deepchem/deepchem/blob/master/.github/workflows/torch_setup.yml#L36) but is running on Linux and Windows only.
- Is there an alternative way to run the tests to pass locally?   

Thanks.

--- a/requirements/torch/env_torch.mac.cpu.yml
+++ b/requirements/torch/env_torch.mac.cpu.yml
@@ -8,3 +8,4 @@ dependencies:
     - torch-scatter
     - torch-sparse
     - torch-geometric
+    - pytorch-lightning 

